packages/bot-web-ui/src/components/quick-strategy/quick-strategy.scss

```diff
@@ -18,6 +18,22 @@
     }
 }
 
+.dc-input .dc-input {
+    &__field {
+        &:focus,
+        &:disabled,
+        &:not(:focus):not([value='']) {
+            outline: none;
+
+            & ~ label {
+                transform: translate(0, -1.8rem) scale(0.875);
+            }
+        }
+    }
+    &__label {
+        font-size: var(--text-size-xxs);
```

Comment:
is it possible to use Text Component instead of css class?

Reply:
unfortunately no. I'm overwriting styles for specific cases of existent component

Reply:
The `label` prop in the `Input` component can support elements inside. The `Text` component allows you to manage specific cases as well.

Reply:
We can also add a prop `label_className` to `Input`.

Reply:
but isn't it more safe and eloquent to change styles where I need it than adding an additional logic to common used component? Nijil asked me to change styles in `quick-strateg.scss` and to change nothing in common used components if it is possible

Reply:
It's okay to add something that can help everyone and this prop will, please add it. It's not much of additional logic, you add the prop and give the prop the className for these cases.

cc: @nijil-binary

Reply:
@Daniil-Deriv It is perfectly alright to add a prop if it's useful. In this case here, `label_className` and `field_className` prop can be added to `Input` without breaking anything and can be used by everyone to pass specific styles for field and label if required.

Reply:
added